hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/TestHoodieFileGroupReaderBasedRecordReader.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.hadoop;
+
+import org.apache.hudi.common.util.collection.ClosableIterator;
+
+import org.apache.hadoop.io.ArrayWritable;
+import org.apache.hadoop.mapred.InputSplit;
+import org.apache.hadoop.mapred.JobConf;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.UnaryOperator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+
+public class TestHoodieFileGroupReaderBasedRecordReader {
+
+  @Test
+  void testGetProgressForLogFilesOnlyInMergeOnReadTable() throws Exception {
+    HoodieFileGroupReaderBasedRecordReader recordReader = new HoodieFileGroupReaderBasedRecordReader(
+        mock(HiveHoodieReaderContext.class),
+        mock(ClosableIterator.class),
+        mock(ArrayWritable.class),
+        mock(InputSplit.class),
+        mock(JobConf.class),
+        mock(UnaryOperator.class),
+        false
+        );
+
+    assertEquals(0, recordReader.getProgress());
```

Comment:
nit: test both `getProgress` and `getPos` in the same test since the test scenario is the same (rename it to `testGetProgressAndPosForLogFilesOnlyInMergeOnReadTable`)?